docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -744,3 +746,5 @@ When I select next value with `0.1` offset in picker wheel located `xpath(//XCUI
 
 
 include::partial$generic-ui-steps.adoc[]
+
+include::partial$proxy-steps.adoc[]
```

Comment:
we should put some extra info, that proxy is not supported by Appium and custom solutions are needed